en/04-licences.md

```diff
@@ -0,0 +1,104 @@
+
+## Free Software & Open Source
+
+The Free Software Foundation (FSF) (www.fsf.org), an American non-profit company founded in 1985 by R. Stallman, supports the GNU operating system and manages "GPL" licenses
+
+This foundation gives a definition of free software based on four irrevocable liberties:
+
+- Freedom to use the software
+- Freedom to copy the software. (includes the freedom to sell copies)
+- Freedom to study the software. (suppose access to the source code)
+- Freedom to modify the software and redistribute the modified versions.
+
+Free license works are not always available for free and associated services (development, warranties, support, etc.) are often paid for. A work contaminated by a free license, which could initially be available only against payment, must be rebroadcast freely.
+
+> « Think Free as Free speech and not Free beer » – R. Stallman
+
+While the Open Source Initiative (OSI) (http://opensource.org/), an American non-profit society founded in 1998 by E. Raymond & B. Perens, promotes open source code
+
+According to OSI, for software to be open source, its license must meet the following 10 criteria:
+
+- Free redistribution.
+- Provision of the source code.
+- Allow derivative works and modifications.
+- Integrity of the source code of the author.
+- No discrimination between individuals or groups.
+- No discrimination between the fields of application.
+- No restriction on the distribution of the license (vs NDA).
+- The license must not be specific to a product.
+- The license must not restrict other software.
+- The license must be technologically neutral.
+
+> « Both terms describe virtually the same category of software. But they represent views based on fundamentally different values. » - R. Stallman
+
+https://twitter.com/OpenSourceOrg/status/1174073039534706688?s=20
+
+
+## Copyright, copyleft and public domain
+
+Before presenting the different types of licenses available, it is necessary to understand the different types of rights applied to works. Here is an explanation, in simple terms, of the 3 major areas of enforcement.
+
+** copyright ** (or copyright) is a right granted to the author of the original work, including the right to authorize or prohibit the publication or distribution of their work. They protect authors from copying or unauthorized sale of their works. Copyright is granted for a limited period of time, after which the work enters the public domain.
```

Comment:
For 'copyright' and the following bold styled parts of the file, you must stick the ** with the first and last character of the word/part of the sentence.
So for example replace ** copyright ** by **copyright**